src/caret/movement.rs

```diff
@@ -175,4 +175,24 @@ impl Editor {
 
         None
     }
+
+    /// Get next WORD forward
+    /// "A WORD consists of a sequence of non-blank characters, separated with
+    /// whitespace.  An empty line is also considered to be a WORD."
+    pub fn _next_word_forward(&self, n: usize) -> Option<usize> {
```

Comment:
Damnit,  sorry, I meant to comment that up.

It is to show the difference in functions for WORD vs word.
I tried writing it as next_WORD_forward but rustc gave me a linting error.

Lemme know if you would like that updated and I can make the change.